java/src/jmri/implementation/AbstractTurnout.java

```diff
@@ -762,6 +790,7 @@ public void provideFirstFeedbackNamedSensor(NamedBeanHandle<Sensor> s) {
         }
 
         _firstNamedSensor = s;
+        firePropertyChange("turnoutFeedbackFirstSensorChange", temp, s);
```

Comment:
I think it would be better to move this call to the end of the method. That way, the initial known state is set before the listeners are called.

Reply:
Thanks, good spot.
There were a few instances of this happening.